tools/release-notes/github.go

```diff
@@ -42,7 +42,12 @@ func (p *project) gitHubGetReleaseNote(number int) (string, error) {
 	for i, line := range body {
 		note, err := gitHubParseReleaseNote(i, line, body)
 		if err == nil {
-			note = fmt.Sprintf("[PR #%d][%s] %s", number, *pr.User.Login, note)
+			if len(p.repoPrettyUrl) > 0 {
+				note = fmt.Sprintf("[PR [#%d](%s/pull/%d)][[%s](https://github.com/%s)] %s", number, p.repoPrettyUrl, number, *pr.User.Login, *pr.User.Login, note)
```

Comment:
not related to this PR, but it would be easier to understand the code after renaming `number` to a meaningful name.

Reply:
done